resources/scripts/routers/ServerRouter.tsx

```diff
@@ -90,50 +90,60 @@ const ServerRouter = ({ match, location }: RouteComponentProps<{ id: string }>)
 
     return (
         <React.Fragment key={'server-router'}>
-            <NavigationBar/>
+            {window.location.href.includes('/console') ?
+                <>
+                </>
+                :
+                <NavigationBar/>
```

Comment:
Replace with `!window.location.href.includes('/console') && <NavigationBar/>`

Reply:
Fixed in https://github.com/pterodactyl/panel/pull/3904/commits/b1ed950cff39e1bbe3ed53e697fe34a9b91e7eb3